cellprofiler/modules/identifyobjectsmanually.py

```diff
@@ -42,23 +46,17 @@ class IdentifyObjectsManually(I.Identify):
 
     def create_settings(self):
         self.image_name = cps.ImageNameSubscriber(
-                "Select the input image", cps.NONE, doc="""
-            Choose the name of the image to display in the object
-            selection user interface.""")
+                "Select the input image", cps.NONE, doc="""Choose the name of the image to display in the object selection user interface.""")
 
         self.objects_name = cps.ObjectNameProvider(
-                "Name the objects to be identified", "Cells", doc="""
-            What do you want to call the objects
-            that you identify using this module? You can use this name to
-            refer to your objects in subsequent modules.""")
+                "Name the objects to be identified", "Cells", doc="""What do you want to call the objects that you identify using this module? You can use this name to
```

Comment:
This docstring is a bit long, please put a `\` after `doc="""` and bump everything after it to the next line.